### DISCUSSION TITLE
Unable to create application when pull helm chart from artifactory

### DISCUSSION THREAD

**Discussion by deadmandrive** [argoproj/argo-cd#8920] [General]
I have ArgoCD Deploy in Kubernetes cluster and try to deploy application using helm chart.
The chart is in Jfrog Artifactory. I add connection to repo and it connected.
But when I try to create app by helm and select Jfrog Artifactory Repo and it Error said Version Not Found.
![image](https://user-images.githubusercontent.com/28984050/160522914-29e9d4d1-c8af-4d9e-b61b-a46221a080c6.png)

And here my chart struture in Artifactory.
![image](https://user-images.githubusercontent.com/28984050/160523074-8b543714-fbe9-4d4f-9019-fa888ae7df1d.png)

Any suggestion would be great.
Thank you.

**Comment by ixxeL2097** [comment]
I have exactly the same setup but different error message from ArgoCD :

```
Unable to create application: application spec for dazd is invalid: InvalidSpecError: Unable to generate manifests in : rpc error: code = Unknown desc = `helm pull --destination /tmp/f8ce1ce6-34a6-4a36-8a78-bcde5cc5e4d7 --version 1.0.101 --username ****** --password ****** --repo https://jfrog.test.com/artifactory/helm-test-virtual all-in-one-alpine` failed exit status 1: Error: failed to fetch https://jfrog.test.com:443/artifactory/api/helm/helm-test-virtual/testing/all-in-one-alpine-1.0.101.tgz : 401
```

My jfrog repository is connected everything seems fine
